src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/LinuxNetworkInterface.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
```

Comment:
I've gone through and removed all unused usings. I'll do this again incrementally so ones from intermediate changes stay out.